### ISSUE TITLE
Cant open socket.

### ISSUE THREAD

**Issue by gspzz** [WietseWind/xrp-trustline-set-commandline#1]
I do get following problem. Somehow it does look like it cant open the socket. Can you please help?

npm@6.14.15
node.js v14.18.0

> C:\Users\gsp\Documents\GitHub\xrp-trustline-set-commandline>node online-with-family-seed.js
subscribe error RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= 0 and <= 4294967295. Received -503173757
    at new NodeError (internal/errors.js:322:7)
    at checkInt (internal/buffer.js:71:11)
    at writeU_Int32BE (internal/buffer.js:801:3)
    at Buffer.writeUInt32BE (internal/buffer.js:814:10)
    at WebSocketFrame.toBuffer (C:\Users\gsp\Documents\GitHub\xrp-trustline-set-commandline\node_modules\websocket\lib\WebSocketFrame.js:257:24)
    at WebSocketConnection.sendFrame (C:\Users\gsp\Documents\GitHub\xrp-trustline-set-commandline\node_modules\websocket\lib\WebSocketConnection.js:857:43)
    at WebSocketConnection.fragmentAndSend (C:\Users\gsp\Documents\GitHub\xrp-trustline-set-commandline\node_modules\websocket\lib\WebSocketConnection.js:793:14)
    at WebSocketConnection.sendUTF (C:\Users\gsp\Documents\GitHub\xrp-trustline-set-commandline\node_modules\websocket\lib\WebSocketConnection.js:733:10)
    at W3CWebSocket.send (C:\Users\gsp\Documents\GitHub\xrp-trustline-set-commandline\node_modules\websocket\lib\W3CWebSocket.js:116:26)
    at C:\Users\gsp\Documents\GitHub\xrp-trustline-set-commandline\node_modules\rippled-ws-client\src\index.js:249:36 {
  code: 'ERR_OUT_OF_RANGE'
}
SignError

 {
  type: 'account_info_error',
  message: 'Error retrieving account_info',
  error: RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= 0 and <= 4294967295. Received -1284492122

**Comment by WietseWind** [comment]
Hey,

A couple of things:
- I never tested this on Windows
- I released this code 4 years ago, a lot has changed since

I'm going to remove this repository, but before I do that I'll wait a couple of days for your reply here. Let's find another method to reach each other, so I can help you proceed with today's libs/tech instead of what I released four years ago. 

Can you describe what you are trying to achieve & how I can reach you to get you more info/samples?

Yours,
Wietse

**Comment by gspzz** [comment]
Hi WietseWind,

I followed the development closely of yours and the XRPL the last 4 years. I did wrote in C many years ago but didnt used it for a while. I wanted to learn more about the XRPL and wanted to learn more about new languages. Python and JS/Node would be great to understand.

My first target was to solve on simple thing in an not GUI Area (Xumm is great, but I wanted to learn more about current XRPL tools and development sets incl. languages): How to set a trustline on my wallet.

Whats the most convenient way for you - to reach out to you?
